### PR TITLE
Correct  some Farsi translation misspelling in locale-fa.json

### PR DESCRIPTION
**Pull request by vaheeds**

### CODE REVIEW

File: app/locales/taiga/locale-fa.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
             "RANGE_CHECK": "می‌بایست بین %s الی %s انتخاب داشته باشید.",
             "EQUAL_TO": "این مقدار می‌بایست یکسان باشد",
             "LINEWIDTH": "احتمالاً یک یا چند خط بیش از حد طولانی است. سعی کنید کمتر از %s حرف داشته باشد.",
-            "PIKADAY": "فرمت تاریخ نامعتبر، لطفاً از فرمت روز ماه سال (مثل 23 مارس 1984) استفاده کنید"
+            "PIKADAY": "فرمت تاریخ نامعتبر، لطفاً از فرمت روز ماه سال (مثل 15 شهریور 1362) استفاده کنید"
```
Comment (vaheeds) [inline app/locales/taiga/locale-fa.json]: Change the example date from gregorian to Persian format.

File: app/locales/taiga/locale-fa.json [hunk 2]
```diff
@@ -143,7 +143,7 @@
             "SLUG": "نام کوتاه",
             "COLOR": "رنگ",
             "IS_CLOSED": "آیا بسته شده؟",
-            "STATUS": "عیت",
```
Comment (vaheeds) [inline app/locales/taiga/locale-fa.json]: This word is incorrect in Persian.

File: app/locales/taiga/locale-fa.json [hunk 3]
```diff
@@ -237,13 +237,13 @@
...
             "CATEGORIES": {
                 "TYPE": "نوع",
-                "STATUS": "عیت",
+                "STATUS": "وضعیت",
                 "SEVERITY": "اهمیت",
                 "PRIORITIES": "اولویت‌ها",
                 "TAGS": "برچسب‌ها",
                 "ASSIGNED_TO": "اختصاص یافته به",
                 "ASSIGNED_USERS": "کاربران اختصاص یافته",
-                "ROLE": "قش",
```
Comment (vaheeds) [inline app/locales/taiga/locale-fa.json]: This word is incorrect too.

**Comment by migonzalvar** [comment]
@vaheeds could you please review the translation at Transifex. Please, visit https://www.transifex.com/taiga-agile-llc/taiga-front/translate/#fa_IR/

Thank you!

**Comment by migonzalvar** [comment]
Some of the strings has been updated in 94b4c1e0c0442ca7fff7ccdcc48e147d87ab446f

**Pull request closed without merging**